src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -893,6 +900,21 @@ private void sendBuildingPackets(@NotNull final Set<EntityPlayerMP> oldSubscribe
     }
 
     /**
+     * Sends packages to update the schematics.
+     *
+     * @param oldSubscribers    the existing subscribers.
+     * @param hasNewSubscribers the new subscribers.
+     */
+    private void sendSchematicsPackets(@NotNull final Set<EntityPlayerMP> oldSubscribers, final boolean hasNewSubscribers)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "oldSubscribers". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1172)